ProjFS.Mac/PrjFSKextLogDaemon/PrjFSKextLogDaemon.cpp

```diff
@@ -199,7 +199,7 @@ static void LogPanics()
                if (panicLogTime == 0)
                {
                    // Record error if we can't determine timestamp
-                   os_log(s_daemonLogger, "Unable to run stat on panic at %s errno=%d strerror=%s\n", fullFileName.c_str(), errno, strerror(errno));
```

Comment:
Is the `strerror` similarly affected, do we need to mark that as public too? If that's not necessary, any idea why not?

Reply:
I would guess that is affected.  In my experience every %s will be private by default.

Reply:
This is what I thought too but there appears to be some kind of heuristic at play which doesn't redact everything. Presumably they run some regexes on the string to guess whether it might be a path, etc.

Reply:
The error messages are not marked as private.  It appears there is a heuristic.  

[This article](https://www.lordcodes.com/posts/clear-and-searchable-logging-in-swift-with-oslog) suggests dynamic string content will be marked as private but that's not what I'm seeing here. 

I'll update the others to be %s{public} as well and verify the strings continue to come thru.  Better safe than sorry.

Reply:
Actually, I wonder if it's the fact that the `strerror()` strings are most likely hardcoded and reside in the constant section of a binary, and the system is detecting this from the address and thus marking it as public automatically. Regardless, I'd explicitly mark them as public in case the heuristic changes in future.